src/simulator_core/entities/assets/asset_abstract.py

```diff
@@ -1,11 +1,28 @@
 from abc import ABC, abstractmethod
+from typing import Dict
+
+from pandas import DataFrame
 
 
 class AssetAbstract(ABC):
+    """Abstract class for Asset"""
+
+    @abstractmethod
+    def set(self, **kwargs) -> None:
+        """Placeholder to set attributes of an asset"""
+        pass
+
+    @abstractmethod
+    def get(self, **kwargs) -> Dict:
```

Comment:
Why return a dict and not a float?

Reply:
Goede vraag! Wat willen we sowieso uit een get statement halen? 

Het zou ook zomaar een string kunnen zijn?

Reply:
Eens met Mike,  return type is nog niet bekend.   Alleen stel ik dan voor om 'Any'  terug te geven en dit later aan te scherpen